try {
    throw Error('Error m8');
    } catch(e) {
      console.log(e);
    }
    
    